### PR TITLE
Typo fix in wordpress/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
1. "to service your wordpress application" ->"to the service of your wordpress application" or "to serve your wordpress application"
2. WordPress Image->WordPress image      keep same format in a table.
3. helm->Helm      It's a proper noun

### CODE REVIEW

File: stable/wordpress/README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Note also if you disable MariaDB per above you MUST supply values for the `exter
 This chart provides support for ingress resources. If you have an
 ingress controller installed on your cluster, such as [nginx-ingress](https://kubeapps.com/charts/stable/nginx-ingress)
 or [traefik](https://kubeapps.com/charts/stable/traefik) you can utilize
-the ingress controller to service your WordPress application.
+the ingress controller to the service of your WordPress application.
```
Comment (javsalgar) [inline stable/wordpress/README.md]: I would use " to serve your WordPress application"

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: stable/wordpress/README.md (hunk 1 above)
Comment (AdamDang) [reply stable/wordpress/README.md]: Great idea! @javsalgar 
It's done.

**Pull request merged**